app/ide-desktop/lib/dashboard/public/index.html

```diff
@@ -0,0 +1,54 @@
+<!--
+  TODO [NP]: https://github.com/enso-org/cloud-v2/issues/345
+  Unify this with the `index.html` in `lib/content`. The same HTML should be used in both places,
+  and we should be launching the IDE the same way in both places. Currently we do not do this.
```

Comment:
This should be part of this PR. I was telling a lot about it. Making so many entry points makes this hard to understand and hard to debug. I do not want ot make the state of this app so fractured and I really do not want to merge the PR until this is fixed.